### PR TITLE
fix: 🐛 Empty check for xyxy results for prevent ValueError and return as empty detection

### PR DESCRIPTION
**Pull request by onuralpszr**
# Description

We have multiple different detection and in case of video or multiple images it can be return empty so prevent that workflow break we should check all other detection type xyxy in case of empty boxes and continue 

List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

-   [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by onuralpszr** [comment]
@hardikdava  I was thinking bit more big before sending PR but anyway here we go could you check all of them please.

File: supervision/detection/core.py [hunk 1]
```diff
@@ -178,6 +178,10 @@ def from_yolov5(cls, yolov5_results) -> Detections:
             ```
         """
         yolov5_detections_predictions = yolov5_results.pred[0].cpu().cpu().numpy()
+
+        if np.asarray(yolov5_detections_predictions[:, :4]).shape[0] == 0:
```
Comment (hardikdava) [inline supervision/detection/core.py:182]: Tested and not needed.

File: supervision/detection/core.py [hunk 2]
```diff
@@ -216,6 +220,10 @@ def from_yolov8(cls, yolov8_results) -> Detections:
             >>> detections = sv.Detections.from_yolov8(result)
             ```
         """
+
```
Comment (hardikdava) [inline supervision/detection/core.py:223]: tested and not needed

**Comment by hardikdava** [comment]
@onuralpszr I just tested and it seems this is not necessary at all. It is automatically handled when you convert output from pytorch to numpy.

File: supervision/detection/core.py (hunk 1 above)
Comment (onuralpszr) [reply supervision/detection/core.py:182]: .numpy() ones not needed okay. I will revert those but leave others then.
```suggestion
        if np.asarray(yolov5_detections_predictions[:, :4]).shape[0] == 0:
```

**Comment by onuralpszr** [comment]
@hardikdava I revert the ones not needed and leave rest and add for sam too.

**Comment by onuralpszr** [comment]
@hardikdava friendly ping :)

**Review by hardikdava — APPROVED** [review]
Everything seems fine to me. 👍

**Comment by SkalskiP** [comment]
@onuralpszr up until now, all of those `.from_*` methods resulted in incorrect `sv.Detections` objects if the model result was empty? Do I understand that correctly?

**Comment by onuralpszr** [comment]
> @onuralpszr up until now, all of those `.from_*` methods resulted in incorrect `sv.Detections` objects if the model result was empty? Do I understand that correctly?

Yes and we switch to empty for prevent "Value Error" in case of no detection

**Pull request merged**